helpdesk_mgmt/controllers/main.py

```diff
@@ -0,0 +1,77 @@
+import base64
+import logging
+
```

Comment:
Delete empty line

Reply:
Are you sure?
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#imports

Reply:
Unable to do it.
Black reformat this code.